src/models/Tag.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * A model for tag in Leto-modelizer.
+ */
+class Tag {
+  /**
+   * Default constructor.
+   * @param {object} [props] - Object that contains all properties to set.
+   * @param {string} [props.type] - The type of this tag.
+   * @param {string} [props.tag] - The value of this tag.
```

Comment:
[props.tag] -> [props.value]
(according to the lines under)

Reply:
done